.github/workflows/django.yml

```diff
@@ -4,17 +4,21 @@ jobs:
   docker_build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - name: Build docker
-      run: docker-compose build
-    - name: Type check
-      run: docker-compose run django mypy --config mypy.ini styleguide_example/
-    - name: Run Flake8
-      run: docker-compose run django flake8
-    - name: Run migrations
-      run: docker-compose run django python manage.py migrate
-    - name: Run tests
-      run: docker-compose run django py.test
+      - uses: actions/checkout@v3
+      - name: Build docker
+        run: docker-compose build
+      - name: Type check
+        run: docker-compose run django mypy --config mypy.ini --cache-dir=/dev/null styleguide_example/
+      - name: Run isort
+        run: docker-compose run django isort styleguide_example/ --check
+      - name: Run black
+        run: docker-compose run django black styleguide_example/ --check
+      - name: Run Flake8
+        run: docker-compose run django flake8
+      - name: Run migrations
+        run: docker-compose run django python manage.py migrate
+      - name: Run tests
+        run: docker-compose run django py.test
```

Comment:
We could sort these by their time of execution.


Example: 
We could put the simpler stuff, like black, isort & flake8 before `mypy` just so that we don't have to wait for the slow stuff(and basically the thing that is most probable to fail) before we execute the stuff that will most-probably always succeed.

My idea is that I'd be bummed if I wait for mypy for 50 seconds(worst case scenario) and then get a flake8 error. 🥲